BridgeSDK/BridgeAPI/SBBBridgeNetworkManager.m

```diff
@@ -163,6 +164,26 @@ - (void)checkForAndHandleUnsupportedAppVersionHTTPError:(NSError *)error
     }
 }
 
+- (void)checkForAndHandleServerPreconditionNotMetHTTPError:(NSError *)error task:(NSURLSessionDataTask *)task responseObject:(id)responseObject retryObject:(APCNetworkRetryObject *)retryObject
+{
+    if (error.code == SBBErrorCodeServerPreconditionNotMet)
+    {
+        // Look to see if the app delegate handles this error.
+        // Note: check conforms to protocol to ensure that the app delegate is intentionally
+        // implementing this method and not coincidentally using the same method signature for something else.
+        id appDelegate = [[UIApplication sharedApplication] delegate];
+        if (![appDelegate conformsToProtocol:@protocol(SBBBridgeAppDelegate)] ||
+            ![appDelegate respondsToSelector:@selector(handleUserNotConsentedError:sessionInfo:networkManager:)] ||
+            ![appDelegate handleUserNotConsentedError:error sessionInfo:responseObject networkManager:self])
+        {
+#if DEBUG
+            // Log the error to the console
+            NSLog(@"User Not Consented error not handled by app delegate:\n%@", error);
+#endif
```

Comment:
app version handler has a fall-through for the case where the app delegate does not handle it. What happens in this case?